.overcommit.yml

```diff
@@ -39,6 +39,8 @@ PreCommit:
   RailsBestPractices:
     enabled: true
     description: 'Checking Rails best practices'
+    include:
```

Comment:
FYI @callaghanc @fddayan @chrisdolendo this is because one of the best practice rules, "no unused methods" requires that the entire scope of the directory is included to check for usage. If you run rails best practices on a subdir, then it doesn't check outside of that dir for usages of methods.

Reply:
👍